Cargo.toml

```diff
@@ -11,18 +11,17 @@ repository = "https://github.com/risc0/risc0-ethereum/"
 
 [workspace.dependencies]
 # Intra-workspace dependencies
+risc0-build-ethereum = { version = "0.8.0-alpha.1", default-features = false, path = "build" }
 risc0-ethereum-contracts = { version = "0.8.0-alpha.1", default-features = false, path = "contracts" }
-risc0-forge-ffi = { version = "0.8.0-alpha.1", default-features = false, path = "ffi" }
 risc0-ethereum-relay = { version = "0.8.0-alpha.1", default-features = false, path = "relay" }
+risc0-forge-ffi = { version = "0.8.0-alpha.1", default-features = false, path = "ffi" }
 
 alloy-primitives = { version = "0.6", default-features = false, features = ["rlp", "serde", "std"] }
 alloy-sol-types = { version = "0.6" }
 anyhow = { version = "1.0.75" }
 bincode = { version = "1.3" }
-bonsai-rest-api-mock = { version = "0.6.1", default-features = false }
-bonsai-sdk = { version = "0.6.1", default-features = false }
-risc0-zkvm = { version = "0.20", default-features = false }
-risc0-zkp = { version = "0.20", default-features = false }
-# NOTE: Using a git rev temporarily to get an unreleased version of risc0-build.
-# Once the referenced commit is in a released version of risc0-build, this will go back to using a version.
-risc0-build = { git = "https://github.com/risc0/risc0", rev = "7f731662", default-features = false }
+bonsai-rest-api-mock = { git = "https://github.com/risc0/risc0", branch = "release-0.21", default-features = false }
+bonsai-sdk = { git = "https://github.com/risc0/risc0", branch = "release-0.21", default-features = false }
+risc0-build = { git = "https://github.com/risc0/risc0", branch = "release-0.21", default-features = false }
+risc0-zkp = { git = "https://github.com/risc0/risc0", branch = "release-0.21", default-features = false }
+risc0-zkvm = { git = "https://github.com/risc0/risc0", branch = "release-0.21", default-features = false }
```

Comment:
as this pr is not for the release but for main, I thought to keep the branch as it gets all the updates rather than the version (that instead should be used for the release). Also, `bonsai-rest-api-mock` has been dropped from the published crates on risc0, so we don't have a published updated version for it.

Reply:
My preference I think is to use `crates.io` here, unless we need unpublished crates (e.g. using a `git` reference for `bonsai-api-mock`, which is a dev dependency seems fine). It doesn't seem, to me, to add value to use `git` references here, but it does add a step to the release process when we need to change all of these references.

Reply:
But happy to disagree and commit here. Up to you.